sysprep/instance_setup.ps1

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+. "..\constants.ps1"
```

Comment:
This is in a subfolder at time of execution, so parent folder is where constants.ps1 is. but what is $CWD for the execution?

Reply:
This should be the path in the package, so in this case the package root (in the include statement they are all at the same level)

To test create a package and install it, to build a package just run goopack against the goospec file:
https://github.com/google/googet/tree/master/goopack